src/pages/Address/styled.tsx

```diff
@@ -133,3 +133,11 @@ export const AddressTransactionsPagition = styled(CommonPagition)`
     margin: 20px 0 0px 0;
   }
 `
+export const AddressEmptyTransactions = styled.div`
```

Comment:
It doesn't seem to have been used

Reply:
OK. Maybe new UI does not have this feature and I will remove it later.

Reply:
https://github.com/nervosnetwork/ckb-explorer-frontend/pull/164/commits/3c70687d2572156969803d13fa042ff3c7c949be